composer.json

```diff
@@ -37,7 +37,7 @@
     "consolidation/annotated-command": "^2.10.2",
     "consolidation/config": "^1.1.0",
     "consolidation/filter-via-dot-access-data": "^1",
-    "consolidation/output-formatters": "^3.3.1",
+    "consolidation/output-formatters": "^3.4",
```

Comment:
This does look useful. After you run composer update, please commit the changes that happen in .scenario directory.